.eleventy.js

```diff
@@ -500,6 +500,9 @@ module.exports = function(eleventyConfig) {
                     conservativeCollapse: true,
                     preserveLineBreaks: true,
                     removeComments: true,
+                    ignoreCustomComments: [
+                        /^\s+ELEVENTYEDGE/
```

Comment:
I think more simple would be:

```suggestion
                        /\<!-- ELEVENTYEDGE --\>/
```

Reply:
If that was the full comment, yes, but an example full comment is: 

`<!--ELEVENTYEDGE_edge "njk" %}EleventyEdgeNunjucksPrecompile:349b99eaaab03aab275b22b91bad65b5a21877cc28a5a6db3f487f3367129c80ELEVENTYEDGE_edge-->`

Reply:
This regex (as things stand doesn't actually work - was meant to open as Draft)